plugins/huggingface/tests/test_huggingface_chat_completion.py

```diff
@@ -40,10 +40,30 @@ def prompt():
     )
 
 
+@pytest.fixture
+def prompt_with_logprobs():
+    return TextPrompt(
+        text="some text prompt",
+        options=SUTOptions(max_tokens=5, temperature=1.0, random="random", top_logprobs=2),
+    )
+
+
 @pytest.fixture
 def sut_request():
     return HuggingFaceChatCompletionRequest(
         messages=[ChatMessage(role="user", content="some text prompt")],
+        logprobs=False,
+        max_tokens=5,
+        temperature=1.0,
+    )
+
+
+@pytest.fixture
+def sut_request_with_logprobs():
+    return HuggingFaceChatCompletionRequest(
+        messages=[ChatMessage(role="user", content="some text prompt")],
+        logprobs=True,
```

Comment:
Given the duplication between these two fixtures, I think it'd be better to skip using fixtures here and just make it a function or method you call from the tests.

Reply:
Updated!